lisa/schema.py

```diff
@@ -1320,6 +1320,12 @@ def reload_requirements(self) -> None:
 class EnvironmentRoot:
     warn_as_error: bool = field(default=False)
     environments: List[Environment] = field(default_factory=list)
+    retry_on_deployment: int = field(
```

Comment:
`retry` should be enough, and add comments above.

Reply:
updated